examples/webapp/webapp.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
+import asyncio
+from datetime import timedelta
+import pathlib
+import sys
+
+
+from datetime import datetime
+
+from yapapi import (
+    Golem,
```

Comment:
Why multiline import?

Reply:
dunno, I'm leaving that decision to black...

Reply:
but indeed, seems strange black didn't convert it back...

Reply:
Black accepts both versions, so the decision is yours, sorry :(
And I think a different decision would be better :)

Reply:
anyway, updated